crates/sargon/src/gateway_api/models/types/response/transaction/preview/transaction_receipt.rs

```diff
@@ -0,0 +1,17 @@
+use crate::prelude::*;
+
+#[derive(
+    Deserialize,
+    Serialize,
+    Clone,
+    PartialEq,
+    Eq,
+    Debug,
+    derive_more::Display,
+    uniffi::Record,
+)]
+#[display("{status}")]
+pub struct TransactionReceipt {
```

Comment:
missing one line doc, would also be great with `HasSampleValues` impl and the standard:

```rust
#[cfg(test)]
mod tests {
    use super::*;

    #[allow(clippy::upper_case_acronyms)]
    type SUT = .... type goes here;

    #[test]
    fn equality() {
        assert_eq!(SUT::sample(), SUT::sample());
        assert_eq!(SUT::sample_other(), SUT::sample_other());
    }

    #[test]
    fn inequality() {
        assert_ne!(SUT::sample(), SUT::sample_other());
    }
}
```